Feature/assinaturas-atualizar-json (#2412)
mercadopago/devsite-docs: guides/subscriptions/subscription-management.en.md

| Type | Description |
|---|---|
| Search subscription | Allows you to search for subscriptions regardless of their status (active, paused, canceled). To do so, send a GET with the necessary parameters to the [/preapproval/search](/developers/en/reference/subscriptions/_preapproval_search/get) endpoint and execute the request.|
| Change amount | This option lets you change the amount of an existing subscription. Send the new amount via `auto_recurring.transaction_amount` and `auto_recurring.currency_id` in a PUT request to the [/preapproval/{id}](/developers/en/reference/subscriptions/_preapproval_plan_id/put) endpoint. |
| Change card of the main payment method | Allows you to change the card linked to an existing subscription. Send a PUT request with the new token in attribute `card_token_id` to the [/preapproval/{id} endpoint](/developers/en/reference/subscriptions/_preapproval_plan_id/put). |

> | Change the secondary payment method | Allows you to add a second payment method to an existing subscription. If it’s a card, send `card_token_id_secondary` and `payment_method_id_secondary`. For other payment methods, only send `payment_method_id_secondary`. Send a PUT to the [/preapproval/{id}](/developers/en/reference/subscriptions/_preapproval_plan_id/put) endpoint to complete the operation. |

Reviewer: ```suggestion
| Change the secondary payment method | Allows you to add a second payment method to an existing subscription. Send a PUT to the [/preapproval/{id}](/developers/en/reference/subscriptions/_preapproval_plan_id/put) endpoint with `card_token_id_secondary` and `payment_method_id_secondary` if the secondary payment method is a card, and only send `payment_method_id_secondary` for any other payment method. |
```

> | Cancel or pause subscription | Allows you to cancel or pause an existing subscription. To do so, send a PUT with the `status` attribute and the `canceled` value to the [/preapproval/{id}](/developers/en/reference/subscriptions/_preapproval_id/put) endpoint and execute the request. To pause a subscription, send a PUT with the `status` attribute and the `paused` value to the [/preapproval/{id}](/developers/en/reference/subscriptions/_preapproval_id/put) endpoint and execute the request. |

Reviewer: ```suggestion
| Cancel or pause subscription | Allows you to cancel or pause an existing subscription. To cancel a subscription, send a PUT with the `status` attribute and the `canceled` value to the [/preapproval/{id}](/developers/en/reference/subscriptions/_preapproval_id/put) endpoint and execute the request. To pause a subscription, send a PUT with the `status` attribute and the `paused` value to the [/preapproval/{id}](/developers/en/reference/subscriptions/_preapproval_id/put) endpoint and execute the request. |
```
